README.md

```diff
@@ -20,12 +20,16 @@ You are expected to have Elixir and Postgres installed.
 
     mix deps.get
     mix ecto.setup
-    mix phoenix.server
+    npm install
```

Comment:
…but we could link to those instructions.

If it's a super common thing to install Phoenix without NPM, it *could* make sense to repeat the instructions here for convenience.